pkg/cmd/flags/capture.go

```diff
@@ -24,6 +25,7 @@ dir:/path/to/dir                         path where tracee will save produced ar
 profile                                  creates a runtime profile of program executions and their metadata for forensics use.
 clear-dir                                clear the captured artifacts output dir before starting (default: false).
 pcap:[single,process,container,command]  capture separate pcap files organized by single file, files per processes, containers and/or commands
+pcap-snaplen:[size, headers or max]      sets length to be captured from each packet's payload (100b, 512b, 1kb, max or headers). 64kb, max, or higher, makes entire payload available.
```

Comment:
snaplen is the same term used by tcpdump, I wanted to keep it because people already knowing tcpdump will understand more easily.